src/seraphis_impl/serialization_demo_utils.cpp

```diff
@@ -321,19 +295,19 @@ void make_serializable_sp_balance_proof_v1(const SpBalanceProofV1 &proof,
 }
 //-------------------------------------------------------------------------------------------------------------------
 void make_serializable_legacy_ring_signature_v4(const LegacyRingSignatureV4 &signature,
-    ser_LegacyRingSignatureV4_PARTIAL &serializable_signature_out)
+    ser_LegacyRingSignatureV4 &serializable_signature_out)
 {
-    make_serializable_clsag(signature.clsag_proof, serializable_signature_out.clsag_proof_PARTIAL);
+    make_serializable_clsag(signature.clsag_proof, serializable_signature_out.clsag_proof);
     serializable_signature_out.reference_set_COMPACT = signature.reference_set;
     indices_to_offsets(serializable_signature_out.reference_set_COMPACT);
 }
 //-------------------------------------------------------------------------------------------------------------------
 void make_serializable_sp_membership_proof_v1(const SpMembershipProofV1 &proof,
-    ser_SpMembershipProofV1_PARTIAL &serializable_proof_out)
+    ser_SpMembershipProofV1 &serializable_proof_out)
 {
     make_serializable_grootle_proof(proof.grootle_proof, serializable_proof_out.grootle_proof);
-    make_serializable_sp_binned_reference_set_v1(proof.binned_reference_set,
-        serializable_proof_out.binned_reference_set_PARTIAL);
+    serializable_proof_out.bin_loci = proof.bin_loci;
+    serializable_proof_out.bin_rotation_factor = proof.bin_rotation_factor;
```

Comment:
Ngl, I didn't spend much time fussing over the details of serialization since I replaces the code anyways in #39. IMO that's the PR where we should focus on compactness of the format